listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Card from "../components/Card";
+
+export type ListenCountCardProps = {
+  listenCount?: number;
+};
+
+const ListenCountCard = (props: ListenCountCardProps) => {
+  const { listenCount } = props;
+
+  return (
+    <>
+      <Card id="listen-count-card">
+        <h4>
+          You have listened to
+          <hr />
+        </h4>
+        <div>
+          <h4>{listenCount || `&nbsp;`}</h4>
```

Comment:
looking at this again, would this not render the number 0?

Reply:
Didnt Get you